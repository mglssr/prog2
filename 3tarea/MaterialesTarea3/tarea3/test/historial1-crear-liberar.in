crearHistorial 01/01/2024
liberarHistorial
Fin
