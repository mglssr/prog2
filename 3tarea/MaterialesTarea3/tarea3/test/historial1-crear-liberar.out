Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Historial creado con exito.
2> Historial liberada con exito.
3> Fin.
